okio/jvm/src/test/java/okio/ThrottlerTest.kt

```diff
@@ -44,74 +42,6 @@ class ThrottlerTest {
     executorService.shutdown()
   }
 
-  @Test fun sourceByteCount() {
```

Comment:
Removing this because it suffers off-by-ones because the new code truncates differently than the old code. (And now flaky because small movements in nanoTime() can change the result from say, 5 bytes to 6 bytes.)